DMA Engine support
DMA engines can do asynchronous data transfers without
involving the host CPU.  Currently, this framework can be
used to offload memory copies in the network stack and
RAID operations in the MD driver.  This menu only presents
DMA Device drivers supported by the configured arch, it may
be empty in some cases.
Depends: HAS_DMA
---
--- /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config.orig	2022-10-31 13:57:59.705695527 -0400
+++ /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config	2022-11-01 16:20:29.705610307 -0400
@@ -2814,7 +2814,6 @@
 # SATA SFF controllers with BMDMA
 #
 CONFIG_ATA_PIIX=y
-# CONFIG_SATA_DWC is not set
 CONFIG_SATA_MV=m
 CONFIG_SATA_NV=m
 CONFIG_SATA_PROMISE=m
@@ -3187,12 +3186,10 @@
 CONFIG_E1000E_HWTS=y
 CONFIG_IGB=m
 CONFIG_IGB_HWMON=y
-CONFIG_IGB_DCA=y
 CONFIG_IGBVF=m
 CONFIG_IXGB=m
 CONFIG_IXGBE=m
 CONFIG_IXGBE_HWMON=y
-CONFIG_IXGBE_DCA=y
 CONFIG_IXGBE_DCB=y
 CONFIG_IXGBE_IPSEC=y
 CONFIG_IXGBEVF=m
@@ -3252,7 +3249,6 @@
 CONFIG_MLXSW_MINIMAL=m
 CONFIG_MLXFW=m
 CONFIG_NET_VENDOR_MICREL=y
-# CONFIG_KS8842 is not set
 # CONFIG_KS8851 is not set
 # CONFIG_KS8851_MLL is not set
 CONFIG_KSZ884X_PCI=m
@@ -3262,7 +3258,6 @@
 CONFIG_MICROSOFT_MANA=m
 CONFIG_NET_VENDOR_MYRI=y
 CONFIG_MYRI10GE=m
-CONFIG_MYRI10GE_DCA=y
 CONFIG_FEALNX=m
 # CONFIG_NET_VENDOR_NI is not set
 CONFIG_NET_VENDOR_NATSEMI=y
@@ -4193,7 +4188,6 @@
 # CONFIG_SERIAL_8250_16550A_VARIANTS is not set
 # CONFIG_SERIAL_8250_FINTEK is not set
 CONFIG_SERIAL_8250_CONSOLE=y
-CONFIG_SERIAL_8250_DMA=y
 CONFIG_SERIAL_8250_PCI=y
 CONFIG_SERIAL_8250_EXAR=m
 CONFIG_SERIAL_8250_CS=m
@@ -6631,7 +6625,6 @@
 # CONFIG_SND_SOC_IMG is not set
 CONFIG_SND_SOC_INTEL_SST_TOPLEVEL=y
 CONFIG_SND_SOC_INTEL_SST=m
-CONFIG_SND_SOC_INTEL_CATPT=m
 CONFIG_SND_SST_ATOM_HIFI2_PLATFORM=m
 # CONFIG_SND_SST_ATOM_HIFI2_PLATFORM_PCI is not set
 CONFIG_SND_SST_ATOM_HIFI2_PLATFORM_ACPI=m
@@ -6656,7 +6649,6 @@
 CONFIG_SND_SOC_INTEL_SOF_MAXIM_COMMON=m
 CONFIG_SND_SOC_INTEL_SOF_REALTEK_COMMON=m
 CONFIG_SND_SOC_INTEL_SOF_CIRRUS_COMMON=m
-CONFIG_SND_SOC_INTEL_HASWELL_MACH=m
 CONFIG_SND_SOC_INTEL_BDW_RT5650_MACH=m
 CONFIG_SND_SOC_INTEL_BDW_RT5677_MACH=m
 CONFIG_SND_SOC_INTEL_BROADWELL_MACH=m
@@ -7591,7 +7583,6 @@
 CONFIG_INFINIBAND_MTHCA=m
 CONFIG_INFINIBAND_MTHCA_DEBUG=y
 CONFIG_INFINIBAND_QIB=m
-CONFIG_INFINIBAND_QIB_DCA=y
 CONFIG_INFINIBAND_CXGB4=m
 CONFIG_INFINIBAND_EFA=m
 CONFIG_INFINIBAND_IRDMA=m
@@ -7763,42 +7754,7 @@
 #
 # CONFIG_RTC_DRV_HID_SENSOR_TIME is not set
 # CONFIG_RTC_DRV_GOLDFISH is not set
-CONFIG_DMADEVICES=y
-# CONFIG_DMADEVICES_DEBUG is not set
-
-#
-# DMA Devices
-#
-CONFIG_DMA_ENGINE=y
-CONFIG_DMA_VIRTUAL_CHANNELS=y
-CONFIG_DMA_ACPI=y
-CONFIG_ALTERA_MSGDMA=m
-CONFIG_INTEL_IDMA64=m
-CONFIG_INTEL_IDXD_BUS=m
-CONFIG_INTEL_IDXD=m
-# CONFIG_INTEL_IDXD_COMPAT is not set
-CONFIG_INTEL_IDXD_SVM=y
-CONFIG_INTEL_IDXD_PERFMON=y
-CONFIG_INTEL_IOATDMA=m
-# CONFIG_PLX_DMA is not set
-CONFIG_AMD_PTDMA=m
-# CONFIG_QCOM_HIDMA_MGMT is not set
-# CONFIG_QCOM_HIDMA is not set
-CONFIG_DW_DMAC_CORE=y
-CONFIG_DW_DMAC=m
-CONFIG_DW_DMAC_PCI=y
-CONFIG_DW_EDMA=m
-CONFIG_DW_EDMA_PCIE=m
-CONFIG_HSU_DMA=y
-# CONFIG_SF_PDMA is not set
-# CONFIG_INTEL_LDMA is not set
-
-#
-# DMA Clients
-#
-CONFIG_ASYNC_TX_DMA=y
-# CONFIG_DMATEST is not set
-CONFIG_DMA_ENGINE_RAID=y
+# CONFIG_DMADEVICES is not set
 
 #
 # DMABUF options
@@ -7815,7 +7771,6 @@
 CONFIG_DMABUF_HEAPS_CMA=y
 # end of DMABUF options
 
-CONFIG_DCA=m
 CONFIG_AUXDISPLAY=y
 CONFIG_CHARLCD=m
 CONFIG_LINEDISP=m
@@ -9793,10 +9748,7 @@
 CONFIG_CRYPTO_DEV_ATMEL_SHA204A=m
 CONFIG_CRYPTO_DEV_CCP=y
 CONFIG_CRYPTO_DEV_CCP_DD=m
-CONFIG_CRYPTO_DEV_SP_CCP=y
-CONFIG_CRYPTO_DEV_CCP_CRYPTO=m
 CONFIG_CRYPTO_DEV_SP_PSP=y
-# CONFIG_CRYPTO_DEV_CCP_DEBUGFS is not set
 CONFIG_CRYPTO_DEV_QAT=m
 CONFIG_CRYPTO_DEV_QAT_DH895xCC=m
 CONFIG_CRYPTO_DEV_QAT_C3XXX=m
